h2o-bindings/bin/gen_java.py

```diff
@@ -637,12 +669,16 @@ def generate_main_class(endpoints):
             try {
               int i = 0;
               for (String id: ids) {
-                keys[i] = clz.newInstance();
+                keys[i] = clz.getConstructor().newInstance();
```

Comment:
Almost equivalent. My IntelliJ says that the `.getConstructor()` version is safer because if there is any exception raised in the constructor, then the original version would gobble them up, while this one won't.

The class `clz` is loaded using standard class loading mechanisms: someone calls for example `stringArrayToKeyArray(array, FrameKeyV3.class)`, and then Java finds that class among the imports... (Or maybe I don't understand your question)